projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.ts

```diff
@@ -24,24 +31,31 @@ export class NgxSkeletonLoaderComponent implements OnInit, AfterViewInit, OnDest
   static ngAcceptInputType_animation: boolean | string;
 
   @Input()
-  count = 1;
+  count: NonNullable<NgxSkeletonLoaderConfig['count']>;
```

Comment:
Now that all properties in `NgxSkeletonLoaderConfig` are required, you don't need `NonNullable<...>`. Just remove this and everything will work as expected :)

Reply:
Oh yeah, since we actually externalized that optional part via the Partial interface! Thank you for the review, I highly appreciate it! :D